<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RSLunchDialog</class>
 <widget class="QDialog" name="RSLunchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>296</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Russian Speaking Lunch</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QGroupBox" name="lunchDaysGroupBox">
     <property name="title">
      <string>Lunch Days</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="mondayCheckBox">
        <property name="text">
         <string>Monday</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QCheckBox" name="wednesdayCheckBox">
        <property name="text">
         <string>Wednesday</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QCheckBox" name="fridayCheckBox">
        <property name="text">
         <string>Friday</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="tuesdayCheckBox">
        <property name="text">
         <string>Tuesday</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="thursdayCheckBox">
        <property name="text">
         <string>Thursday</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="editelementsHorizontalLayout">
     <item>
      <layout class="QGridLayout" name="editElementsGridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="launchTimeLabel">
         <property name="text">
          <string>Lunch Time:</string>
         </property>
         <property name="buddy">
          <cstring>lunchTimeEdit</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QTimeEdit" name="lunchTimeEdit">
         <property name="time">
          <time>
           <hour>12</hour>
           <minute>0</minute>
           <second>0</second>
          </time>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="iconLabel">
         <property name="text">
          <string>Icon:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="iconComboBox">
         <item>
          <property name="text">
           <string>Random</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Russia</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/images/russia.png</normalon>
           </iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ukraine</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/images/ukraine.png</normalon>
           </iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Belarus</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/images/belarus.png</normalon>
           </iconset>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Moldova</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/images/moldova.png</normalon>
           </iconset>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="dimaModeLabel">
         <property name="text">
          <string>Dima Mode:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="dimaModeComboBox">
         <item>
          <property name="text">
           <string>Off</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>ON BLYAT'</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="launchAutoLabel">
         <property name="text">
          <string>Launch automatically</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="launchAutoComboBox">
         <item>
          <property name="text">
           <string>No</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Yes</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="editElementsHorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsHorizontalLayout">
     <item>
      <spacer name="buttonsHorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>mondayCheckBox</tabstop>
  <tabstop>tuesdayCheckBox</tabstop>
  <tabstop>wednesdayCheckBox</tabstop>
  <tabstop>thursdayCheckBox</tabstop>
  <tabstop>fridayCheckBox</tabstop>
  <tabstop>lunchTimeEdit</tabstop>
  <tabstop>iconComboBox</tabstop>
  <tabstop>dimaModeComboBox</tabstop>
  <tabstop>launchAutoComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RSLunchDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>237</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RSLunchDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>69</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
